lvsfunc/misc.py

```diff
@@ -408,6 +410,53 @@ def upscale_chroma(n: int, f: vs.VideoFrame) -> vs.VideoNode:
     return cast(F, inner)
 
 
+def colored_clips(amount: int, max_hue: int = 300, **kwargs: Any) -> List[vs.VideoNode]:
+    """
+    Returns a list of BlankClips with randomized unique colors. Useful maybe for comparison functions
+    or just for getting multiple uniquely colored BlankClips for testing purposes.
+
+    `max_hue` is the maximum hue that will be returned from the HSL color model.
+
+    Written by Dave <orangechannel@pm.me>.
+
+    :param amount:  Number of `vapoursynth.VideoNode`s to return
+    :param max_hue: Maximum hue (0 < hue <= 360) in degrees to generate colors from (Default: 300)
+    :param kwargs:  Arguments passed to `vapoursynth.core.std.BlankClip` (Default: keep=1)
+
+    :return:        List of uniquely colored clips in random order.
+    """
+    if amount < 2:
+        raise ValueError("colored_clips: `amount` must be at least 2")
+    if not (0 < max_hue <= 360):
+        raise ValueError("colored_clips: `max_hue` must be greater than 0 and less than 360 degrees")
+
+    blank_clip_args = dict(keep=1)
+    blank_clip_args.update(kwargs)
+
+    diff = max_hue / (amount - 1)
+    hues: List[Union[float, int]] = [0]
+
+    for _ in range(amount - 2):
+        hues.append(hues[-1] + diff)
+
+    hues.append(max_hue)
+    hues = [f / 360 for f in hues]
+    hls_color_list: List[List[float]] = []
+
+    for hue in hues:
+        hls_color_list.append(list(colorsys.hls_to_rgb(hue, .5, 1)))
+
+    rgb_color_list: List[List[int]] = [[int(f * 255) for f in color] for color in hls_color_list]
```

Comment:
if this has the possibility of overflowing an 8bit color then just do `min(int(f * 255), 255)` here and remove lines 452-455
Edit: doesn't seem like this is a possibility so this can be left as-is and the other lines removed